acceptance/bundle/deploy/volume/recreate/test.toml

```diff
@@ -2,7 +2,7 @@ Local = false
 Cloud = true
 RequiresUnityCatalog = true
 
-EnvMatrix.DATABRICKS_CLI_DEPLOYMENT = ["terraform"]  # volumes are not supported
+EnvMatrix.DATABRICKS_CLI_DEPLOYMENT = ["terraform"]  # needs volume recreation, grants
```

Comment:
this comment is related to the fact that direct-exp is not included into matrix? I'd add it to the comment, because out of the context this comment does not make a lot of sense to the reader

Reply:
Yes, you guessed right. There are quite a few such tests and I'm adding a short reminder to each case.